ztp/policygentool/kustomize-plugins-locally.md

```diff
@@ -0,0 +1,32 @@
+# **Run PolicyGenTool Kustomize Plugins Locally**
+
+This document covers how to run the [PolicyGenTool Generator plugins](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp) locally.
+
+1. Get Kustomize and install it, you can get it from the [releases page](https://github.com/kubernetes-sigs/kustomize/releases/).
+2. Extract plugins from the PolicyGenTool image:
+
+    ~~~sh
+    mkdir /tmp/ztp-kustomize/
+    podman cp $(podman create --name policgentool --rm quay.io/redhat_emp1/ztp-site-generator:4.10.0-1):/kustomize /tmp/ztp-kustomize/
```

Comment:
```suggestion
mkdir -p ~/.config/kustomize/plugin
podman cp $(podman create --name policgentool --rm quay.io/redhat_emp1/ztp-site-generator:4.10.0-1):/kustomize/plugin/ran.openshift.io ~/.config/kustomize/plugin/
```

Reply:
Not sure if I'm missing something but maybe you can skip the /tmp/ stuff and copy the files directly (?)

Reply:
You're right. Committed the suggestion.